pkg/webhooks/admission/jobs/validate/admit_job.go

```diff
@@ -204,6 +210,17 @@ func validateJobCreate(job *v1alpha1.Job, reviewResponse *admissionv1.AdmissionR
 	if totalReplicas < job.Spec.MinAvailable {
 		msg += " job 'minAvailable' should not be greater than total replicas in tasks;"
 	}
+	if job.Spec.MinAvailable != 0 && !hasSetTaskMinAvailable {
+		if len(job.Spec.Tasks) > 1 && job.Spec.MinAvailable != totalReplicas {
+			msg += " Multi-tasks job 'minAvailable' needs to set task 'minAvailable', " +
+				"otherwise job 'minAvailable' must be equal to the sum of task 'replicas';"
+		}
```

Comment:
I don't think this is needed

Reply:
If
job.Spec.MinAvailable == 5 , totalReplicas =6 ,I think this is OK

Reply:
I did not explain my idea. If multi-task does not set task.minAvailable, it cannot explain that those tasks are equal to job.minAvailable.
```yaml
apiVersion: batch.volcano.sh/v1alpha1
kind: Job
metadata:
  name: minavailable-job
spec:
  schedulerName: volcano
  minAvailable: 2
  tasks:
    - replicas: 3
      name: "master"
      template:
        metadata:
          name: master
        spec:
          containers:
            - image: nginx
              name: nginx
              resources:
                requests:
                  cpu: "1"
                  memory: "1Gi"
          restartPolicy: OnFailure
    - replicas: 3
      name: "work"
      template:
        metadata:
          name: web
        spec:
          containers:
            - image: nginx
              name: nginx
              resources:
                requests:
                  cpu: "1"
                  memory: "1Gi"
          restartPolicy: OnFailure
```
master Running 2 ? work Running 2 ? also master 1 and work 1 is ok?